src/cdrizzleapi.c

```diff
@@ -23,425 +23,447 @@
 static PyObject *gl_Error;
 FILE *driz_log_handle = NULL;
 
-/** --------------------------------------------------------------------------------------------------
+/** ---------------------------------------------------------------------------
  * Multiply each pixel in an image by a scale factor
  */
 
 static void
-scale_image(PyArrayObject *image, double scale_factor) {
-  long  i, size;
-  float *imptr;
+scale_image(PyArrayObject *image, double scale_factor)
+{
+    long i, size;
+    float *imptr;
 
-  assert(image);
-  imptr = (float *) PyArray_DATA(image);
+    assert(image);
+    imptr = (float *)PyArray_DATA(image);
 
-  size = PyArray_DIMS(image)[0] * PyArray_DIMS(image)[1];
+    size = PyArray_DIMS(image)[0] * PyArray_DIMS(image)[1];
 
-  for (i = size; i > 0; --i) {
-    *imptr++ *= scale_factor;
-  }
+    for (i = size; i > 0; --i) {
+        *imptr++ *= scale_factor;
+    }
 
-  return;
+    return;
 }
 
-/** --------------------------------------------------------------------------------------------------
+/** ---------------------------------------------------------------------------
  * Top level function for drizzling, interfaces with python code
  */
 
 static PyObject *
 tdriz(PyObject *obj UNUSED_PARAM, PyObject *args, PyObject *keywords)
```

Comment:
Long functions should be broken up into smaller ones.  An attempt should be made to keep functions small enough to be able to see the whole function without needing to scroll.  Historically, that's been assumed to be about 35 lines long.

Reply:
I do not think this is part of a formatting style and I do not know what to think about it. I have no issues with functions being longer than 35 lines.